Modules/binascii.c

```diff
@@ -334,14 +334,15 @@ binascii_a2b_uu_impl(PyObject *module, Py_buffer *data)
 binascii.b2a_uu
 
     data: Py_buffer
-    /
```

Comment:
While I support keeping *data* positional-only, but in bpo-25357 `b2a_base64` was changed and as a side effect *data* is able to accept a keyword argument. The consistency that functions in module binascii all support a positional-only *data* was broken. What's your mind Serhiy and Martin? We keep *data* positional-only or just make it like `b2a_base64` or make `b2a_base64` *data* back?